package.json

```diff
@@ -1561,25 +1554,24 @@
                     "type": "number",
                     "default": 60000,
                     "description": "%jupyter.configuration.jupyter.jupyterLaunchTimeout.description%",
-                    "scope": "resource"
+                    "scope": "application"
```

Comment:
I don't see why this would be different for each workspace. hence changed to be per a user setting